summary/percentiles.go

```diff
@@ -147,23 +148,30 @@ func getPercentComplete(stats []*secondSample) (percent int32) {
 	return
 }
 
+// Calculate cpurate from two consecutive total cpu usage samples.
+func getCpuRate(latest, previous secondSample) (uint64, error) {
+	var elapsed int64
+	elapsed = latest.Timestamp.Sub(previous.Timestamp).Nanoseconds()
+	if elapsed < 10*milliSecondsToNanoSeconds {
+		return 0, fmt.Errorf("elapsed time too small: %d ns: time now %s last %s", elapsed, latest.Timestamp.String(), previous.Timestamp.String())
+	}
+	cpuRate := (latest.Cpu - previous.Cpu) * secondsToMilliSeconds / uint64(elapsed)
```

Comment:
Shouldn't this just be `/ uint64(elapsed)` The difference in CPU is in nanoseconds and we'll divide by the interval (in ns) to get the number of cores used in that interval.